plugins/generator-1.21.1/neoforge-1.21.1/procedures/utils/double_from_text.java.ftl

```diff
@@ -0,0 +1,6 @@
+private static double doubleFromText(String s) {

```

Comment:
Those changes are completely unrelated to the origianl PR, please split this into another PR..

Reply:
Please read https://github.com/MCreator/MCreator/blob/master/CONTRIBUTING.md